.github/workflows/test.yml

```diff
@@ -12,25 +12,24 @@ jobs:
     strategy:
       matrix:
         go:
-          - "1.22.x"
+          - "oldstable"
```

Comment:
i'm sure this is fine, but can you educate me on what "oldstable" does?

technically the readme claims that you need "go 1.21" or later, but clearly we're not testing that.

Reply:
stable is the latest version of go
oldstable is the previous version 

basically right now this would test 1.24 and 1.23 happy to change this

Reply:
Okay annoyingly the make file requires go1.24 so I have to hard code that for the format task.

I also updated the README to mention 1.22 since that is what is tested.